model/patch_lifecycle.go

```diff
@@ -107,13 +107,16 @@ func AddNewTasksForPatch(ctx context.Context, p *patch.Patch, patchVersion *Vers
 	return errors.Wrap(err, "can't add new tasks")
 }
 
-// IncludePatchDependencies takes a project and a slice of variant/task pairs names
+// IncludeDependencies takes a project and a slice of variant/task pairs names
 // and returns the expanded set of variant/task pairs to include all the dependencies/requirements
 // for the given set of tasks.
 // If any dependency is cross-variant, it will include the variant and task for that dependency.
-func IncludePatchDependencies(project *Project, tvpairs []TVPair) []TVPair {
+func IncludeDependencies(project *Project, tvpairs []TVPair, requester string) []TVPair {
```

Comment:
Since this is more general now, I would move it to the model package and remove the default requester. The actual logic is still called patch_dependencies which isn't accurate but that can be moved later

Reply:
I mostly wanted a default requester so if i missed a case I don't break the world 😓  but you're right it's probably not necessary.

Reply:
I'll just rename the file yolo